common/metrics.py

```
@@ -0,0 +1,66 @@
+    ["cache_prefix", "status"],  # status: success|error
+)
+
+
+class _MetricsFacade:
+    """Thin facade used throughout the app to record cache metrics.
+
+    Methods are intentionally minimal and stable so callers aren't tied to
+    prometheus_client specifics.
+    """
+
+    def increment_cache(self, cache_prefix: str, event: str, reason: str | None = None):
```

Ruff (0.12.2)

Missing return type annotation for private function `increment_cache`

Add return type annotation: `None`

(ANN202)